src/main/scala/uk/gov/hmrc/slugbuilder/SlugBuilder.scala

```diff
@@ -111,4 +129,11 @@ class SlugBuilder(
       CREATE_NEW
     )
 
+  private def copyFiles(fileCSV: Option[String], targetDirectory: Path): String =
```

Comment:
Very minor suggestion, but naming of fileCSV and fileCsv very similar, maybe fileCSV could be maybeFileCsv ? Only a suggestion.